tensorflow_addons/layers/python/normalizations.py

```diff
@@ -0,0 +1,284 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+# Orginal implementation from keras_contrib/layer/normalization
+
+from tensorflow.keras.layers import Layer, InputSpec
+from tensorflow.keras import initializers, regularizers, constraints
+from tensorflow.keras import backend as K
+from tensorflow.keras.utils import get_custom_objects
+from tensorflow.python.ops import nn
+
+class GroupNorm(Layer):
```

Comment:
What do you think about changing the name to `GroupNormalization`, which would be consistent with [`tf.keras.layers.BatchNormalization`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/normalization.py#L774)?

Reply:
Agreed. Should be consisten as LayerNormalization GroupNormalization, etc.

Reply:
Funny. This was my first intention to.  I changed it back to match the tf contrib naming